staging/src/k8s.io/apiserver/pkg/util/logs/logs.go

```diff
@@ -70,8 +70,8 @@ func NewLogger(prefix string) *log.Logger {
 	return log.New(KlogWriter{}, prefix, 0)
 }
 
-// GlogSetter is a setter to set glog level.
-func GlogSetter(val string) (string, error) {
+// KlogSetter is a setter to set klog level.
+func KlogSetter(val string) (string, error) {
```

Comment:
I was concerned about doing this functions since it was an exported function and if anyone was referencing this function outside of `kubernetes/kubernetes` would have a breaking change. Not sure if how best to treat this. Also check out my other comment. :)

Reply:
ok，I will close this PR

Reply:
So are we going to close the PR? If not I think this can be handled by adding a release note. The K8s deprecation policy (https://kubernetes.io/docs/reference/using-api/deprecation-policy/) does not address code apis just resource apis. As such I believe we can make this sort of change but I would release note it.